### PR TITLE
[Technical] Add logging and secret key

### PR DESCRIPTION
**Pull request by veekram**
Some changes on README as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ This application is used for building rails API apps.
 
     bundle exec rails c
 
-### Services (job queues, cache servers, search engines, etc.)
+### Services
+    - Job Queues
```
Comment (kelishrestha) [inline README.md]: Lets add these only when it is added.

File: config/secrets.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+development:
```
Comment (kelishrestha) [inline config/secrets.yml]: Lets have a default configuration as well. May be like this:

```
defaults: &defaults
  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

File: config/secrets.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+  secret_key_base: 4a4f7d142c02e4bcff63970123bb281df74ee5def2e591ab477e7ce65d6b01fee5e55bd5b1bec44e4d2ebe94e806aa3bef858519bdefc31545dfbaf8a0c5c570
```
Comment (kelishrestha) [inline config/secrets.yml]: Add EOF.

**Review by kelishrestha — CHANGES_REQUESTED** [review]

**Comment by kelishrestha** [comment]
@veekram Can we put some more details in PR description?

File: config/secrets.yml (hunk 3 above)
Comment (veekram) [inline config/secrets.yml]: Yeah, better. I will back with up that .

**Pull request closed without merging**